### ISSUE TITLE
Colour Contrast - Manual Assessment Checklist page

### ISSUE THREAD

**Issue by irwink** [wet-boew/web-reporting#33]
Manual Assessment Checklist page
http://publiservice.tbs-sct.gc.ca/ws-nw/wr-rw/editor/accessibility-accessibilite/checklist-grille.aspx?lang=eng
Text inside Sample Type box  is hard to read and fails colour contrast.
Text colour #ACA899, background colour #FFFFFF, contrast ratio 2.38:1
